mjcf_to_sdformat/tests/test_add_mjcf_worldbody_to_sdf.py

```diff
@@ -131,6 +149,41 @@ def test_add_model(self):
         assert_allclose([0, 0, 0],
                         su.vec3d_to_list(collision_3.raw_pose().rot().euler()))
 
+        link_4 = model.link_by_index(3)
+        self.assertEqual("body3", link_4.name())
+        self.assertEqual(1, link_4.visual_count())
+        self.assertEqual(1, link_4.collision_count())
+
+        mass_matrix = link_4.inertial().mass_matrix()
+        self.assertEqual(1, mass_matrix.mass())
+        self.assertEqual([0, 0, 0],
+                         su.vec3d_to_list(link_4.inertial().pose().pos()))
+        self.assertEqual([0, 0, 0],
+                         su.vec3d_to_list(link_4.inertial().pose().rot()))
+        assert_allclose([1, 1, 1],
+                        su.vec3d_to_list(mass_matrix.diagonal_moments()))
+        assert_allclose([0, 0, 0],
+                        su.vec3d_to_list(mass_matrix.off_diagonal_moments()))
+
+        assert_allclose([0, 0, 1], su.vec3d_to_list(link_4.raw_pose().pos()))
```

Comment:
Also check it's `relative_to`?

Reply:
https://github.com/gazebosim/gz-mujoco/pull/47/commits/0308cab0f57918d62a8abf540c958dc49bc149d5